config/default.yml

```
@@ -54,6 +54,14 @@ Rails/ActiveSupportAliases:
   Enabled: true
   VersionAdded: '0.48'
 
+Rails/AfterCommitOverride:
+  Description: >-
+                  This cop enforces that there is only one call to `after_commit`
+                  (and its aliases - `after_create_commit`, `after_update_commit`,
+                  and `after_destroy_commit`) with the same callback name per model.
+  Enabled: 'pending'
+  VersionAdded: '2.7'
```

This cop will be introduced in RuboCop Rails 2.8 (next release).

```suggestion
  VersionAdded: '2.8'
```